### ISSUE TITLE
Source glob fails in 0.8.0

### ISSUE THREAD

**Issue by ggmartinsen** [gulp-elm/gulp-elm#37]
Works: gulp.src('../admin-elm/src/users/Users.elm') (Manual array also works)
Fails: 
```
gulp.src('../admin-elm/src/**/*.elm')
.pipe(elm({
            cwd: '../admin-elm/',
            optimize: true
        }))
.pipe(gulp.dest('public/'))

-----
{ Error: ENOENT: no such file or directory, open '/tmp/gulp-elm-tmp-11884-29168-vn8ssn.9eyka/11884-29168-1fb90pi.lblx.js'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/tmp/gulp-elm-tmp-11884-29168-vn8ssn.9eyka/11884-29168-1fb90pi.lblx.js' }

<removed path>.../node_modules/q/q.js:155
                throw e;
                ^
Error: ENOENT: no such file or directory, open '/tmp/gulp-elm-tmp-11884-29168-vn8ssn.9eyka/11884-29168-1fb90pi.lblx.js'
```

**Comment by bdukes** [comment]
Elm 0.19 no longer produces any output for files that don't contain a `main` program.  I'm planning to have _gulp-elm_ not throw an `Error` in that case, but as a workaround, can you adjust the glob to only include the main entry point(s), e.g. `'../admin-elm/src/**/Main.elm'`?

**Comment by ggmartinsen** [comment]
Yep, can just do it manually for now. Just figured I'd let you know in case this was new.

**Comment by bdukes** [comment]
@ggmartinsen I just published 0.8.1 which should fix your issue.  Hope it helps!
